docs/guides/01-overview.md

```diff
@@ -0,0 +1,173 @@
+## Overview
+
+CARTO.js lets you create custom location intelligence applications that leverage the power of the CARTO Platform.
+
+### Audience
+
+This documentation is designed for people familiar with JavaScript programming and object-oriented programming concepts. You should also be familiar with Leaflet from a user's point of view.
+
+This conceptual documentation is designed to let you quickly start exploring and developing applications with the CARTO.js library. We also publish the [CARTO.js API Reference]({{site.cartojs_docs}}/reference/).
+
+### Hello, World
+
+The easiest way to start learning about the CARTO.js library is to see a simple example. The following web page displays a map adding a layer over it.
+
+```html
+      <!DOCTYPE html>
+<html>
+  <head>
+    <title>Single layer | CARTO</title>
+    <meta name="viewport" content="initial-scale=1.0">
+    <meta charset="utf-8">
+    <!-- Include Leaflet -->
+    <script src="https://unpkg.com/leaflet@1.3.1/dist/leaflet.js"></script>
```

Comment:
@ivanmalagon do we have a cartocdn.com link for leaflet? Could we get one? Seems odd and off-brand to do these examples with unpkg...

Reply:
We don't and I think we shouldn't. It implies maintenance in our part, whereas `unpkg` works using the `npm` registry and if something happens with `npm` or the `leaflet`package they'll fix it.

Reply:
👍

Reply:
From https://unpkg.com/#/about

"unpkg is a free, best-effort service and cannot provide any uptime or support guarantees....The goal of unpkg is to provide a hassle-free CDN for npm package authors. It's also a great resource for people creating demos and instructional material. However, if you rely on it to serve files that are crucial to your business, you should probably pay for a host with well-supported infrastructure and uptime guarantees."

Maybe this is a larger issue that should be discussed elsewhere (what is the best place?), but since our customers do use CARTO and supporting libraries in business-crucial scenarios, we should think about either putting these files in a CDN we control, or at the very least documenting with a "disclaimer" of sorts the possible disadvantage of using unpkg in production. Customers will look to the Developer Center as the guide for what they should do in production, and unpkg seems to recommend against using themselves for that.

Reply:
Good points, @andrewbt. I understand that we manage a different policy for CARTO resources (CDN policy) and external resources (it depends a lot on the specific resource). I agree with you that since users are expecting for ready-for-production content, we should add a "disclaimer" explaining that management of external resources.

I think we could add this to the `quickstart` guide as a general note. At the same time, the comment in the code `Include Leaflet` could be extended with the same disclaimer, so that users can read it in the same place the resources are being called. Even maybe a note on the `support` section, as an FAQ, would be worthy.

What do you think?

Reply:
It's a larger issue, yes.

We can start the conversation in [docs repo](https://github.com/CartoDB/docs) since this will be reflected mostly in docs and engage @rochoa to provide the eagle-eye vision on this particular issue on where to host external libraries.

Reply:
@andrewbt, what Leaflet versions do our customers request to use?

Reply:
@andrewbt, the _latest stable release_ term is something that changes over time, so we will need to start hosting every single release. Most serious developers will bundle, manage, and host their JS. As you say, unpkg is what Leaflet website recommends, and for demo purposes, it should be enough.

We have had our internal discussions about using unpkg for CARTO VL demos. However, as we have our `cartocdn` URLs, we opted to only use cartocdn to, precisely, avoid what you mentioned: "_unpkg is a free, best-effort service and cannot provide any uptime or support guarantees_".

Anyway, as @inigomedina proposes, something to consider, decide, fix, and document properly.